source/_docs/mobile-redirection.md

```diff
@@ -45,7 +45,7 @@ In order for CloudFlare's mobile redirection service to work, the domain must be
 
 
 ### JavaScript
-You can detect the page width or user agent via JavaScript, then redirect requests to your configured and optimized mobile site. We do not recommend using cookies that are passed to the backend for mobile theme detection and configuration. For details, see [Varnish Caching - Drupal and WordPress Advanced Topics](/docs/caching-advanced-topics/#device-detection).
+You can detect the page width or user agent via JavaScript, then redirect requests to your configured and optimized mobile site. We do not recommend using cookies that are passed to the backend for mobile theme detection and configuration. For details, see [Caching: Advanced Topics](/docs/caching-advanced-topics/#device-detection).
```

Comment:
As a developer I cringed anytime someone would check page width with JS. There are tons of known issues with this. Page widths don't calculate reliably with JS. If they want to do page width settings, they should be doing responsive design and doing it completely in css. Checking for user agent seems different though